lib/utils.pm

```diff
@@ -101,6 +102,10 @@ sub unlock_if_encrypted {
     }
 }
 
+sub turn_off_packagekit() {
+    x11_start_program("systemctl stop packagekit.service");
```

Comment:
How can this even work? doesn't `systemctl` need to be executed as root? Or are you root by default on live installations?

Also I agree with anton, why not just use an already existing function?